hs_core/tests/api/native/test_bagit.py

```diff
@@ -28,6 +28,11 @@ def setUp(self):
             'My Test Resource'
             )
 
+    def tearDown(self):
+        super(TestBagIt, self).tearDown()
+        delete_resource(self.test_res.short_id)
+        GenericResource.objects.all().delete()
```

Comment:
In this case, only GenericResource is created and used, so GenericResource.objects.all().delete() should suffice, right? What is the benefit of using BaseResource.objects.all().delete() over GenericResource.objects.all().delete()?